src/main/java/com/paneedah/mwc/RecipeManager.java

```diff
@@ -12,9 +13,9 @@ public class RecipeManager {
 
     public static void init(ModContext modContext) {
 
-        compatibility.addSmelting(Ores.CopperOre, new ItemStack(Ores.CopperIngot), 5f);
-        compatibility.addSmelting(Ores.LeadOre, new ItemStack(Ores.LeadIngot), 5f);
-        compatibility.addSmelting(Ores.TinOre, new ItemStack(Ores.TinIngot), 5f);
+        compatibility.addSmelting(Blocks.copper_ore, new ItemStack(Ores.CopperIngot), 5f);
+        compatibility.addSmelting(Blocks.lead_ore, new ItemStack(Ores.LeadIngot), 5f);
+        compatibility.addSmelting(Blocks.tin_ore, new ItemStack(Ores.TinIngot), 5f);
```

Comment:
Need to revert that this isn't the Minecraft id this is just the variable should follow Java convention

Reply:
✅